### DISCUSSION TITLE
https://theme-hope.vuejs.press/zh/guide/markdown/hint.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#759] [Announcements]
# https://theme-hope.vuejs.press/zh/guide/markdown/hint.html

主题可以为你添加提示、注释、信息、注意、警告和详情提示容器的支持。

https://theme-hope.vuejs.press/zh/guide/markdown/hint.html

**Comment by xzy111222333** [comment]
太漂亮了
